dlt/destinations/snowflake/configuration.py

```diff
@@ -0,0 +1,74 @@
+from typing import Final, Optional, Any, Dict
+
+from sqlalchemy.engine import URL
+
+from dlt.common.typing import TSecretStrValue
+from dlt.common.configuration.specs import ConnectionStringCredentials
+from dlt.common.configuration import configspec
+from dlt.common.destination.reference import DestinationClientDwhConfiguration
+
+
+def _read_private_key(private_key: str, password: Optional[str] = None) -> bytes:
+    """Load an encrypted or unencrypted private key from string.
+    """
+    from cryptography.hazmat.backends import default_backend
```

Comment:
OK I see there's no other way to decrypt and parse pkey. Let's throw the usual `MissingDependencyException` with some explanation what we need and why

Reply:
Raising now with the "install dlt[snowflake]" message, this is a snowflake dependency.